change/@uifabric-merge-styles-2020-01-09-23-35-38-csp.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Add note to merge-styles readme about CSP",
```

Comment:
```suggestion
  "comment": "Documentation: Added note to merge-styles readme about working with CSP.",
```

Reply:
I think expanding CSP into its full form would be helpful since not everyone that looks at this release notes may know what CSP stands for - took me a little while to realize it's Content Security Policy & I also had to look up what a nonce is w.r.t content security policies

Reply:
Belatedly replying to this--I didn't want to go into detail here because I think this is the type of thing that's not relevant in the majority of cases, and if it is relevant in your case, you're probably already familiar with the concepts involved and just looking for how you set it up in Fabric.